leaderelection/context_test.go

```diff
@@ -1,6 +1,3 @@
-// +build !race
-// TODO(https://github.com/kubernetes/kubernetes/issues/90952): Remove the above.
```

Comment:
We should delete this too: https://github.com/knative/pkg/blob/ed076316cbe1107590822f930f4ed20b109290f1/test/presubmit-tests.sh#L48

Reply:
@vagababov 👀

Reply:
Sorry :| 
Somehow I missed this.
😢